src/crate/client/tests.py

```diff
@@ -335,6 +312,10 @@ def _try_execute(cursor, stmt):
     try:
         cursor.execute(stmt)
     except Exception:
+        # FIXME: Why does this croak on statements like ``DROP TABLE cities``?
+        # Note: When needing to debug the test environment, you may want to
+        #       enable this logger statement.
+        # log.exception("Executing SQL statement failed")
```

Comment:
### About
Tests: Add inline comment about how to debug the test environment.

### Background story
It was needed because the tests suddenly started failing mysteriously on my machine.

Out of the blue, the test suite started croaking like `TABLE 'locations' already exists`, only to find that CrateDB would not accept any write operations after finding that my disk usage was 96%, while I had still 60G of free disk space.

Unfortunately, I did not discover this easily, because the test suite swallowed the exception on this spot. When enabling the `log.exception` statement, the error response from CrateDB became immediately visible.